src/scilpy/cli/scil_tracking_local_dev.py

```diff
@@ -156,9 +156,20 @@ def _build_arg_parser():
                         'Region-Adaptive Propagation tractography will start within '
                         'this mask.')
     track_g.add_argument('--rap_method', default='None',
-                        choices=['None', 'continue'],
-                        help="Region-Adaptive Propagation tractography method "
+                        choices=['None', 'continue', 'quack'],
+                        help="Region-Adaptive Propagation tractography method." \
+                        "To use option quack, you must install Quacktography " \
+                        "continue : goes straight in the RAP mask, " \
+                        "quack : uses a graph solution by quantum approach. " \
                         " [%(default)s]")
+    track_g.add_argument('--reps', type=int, default=2,
+                         help='Number of repetitions for the RAP method. '
+                             'Default: 2. This is only used for the quack '
+                             'method, not for continue.')
+    track_g.add_argument('--alpha', type=float, default=1.5,
+                         help='Alpha parameter for the RAP method. '
+                              'Default: 1.5. This is only used for the quack '
+                              'method, not for continue.')
 
```

Comment:
For a nicer, view, you could place the three RAP arguments into an argument group.
`p.add_argument_group('RAP options')`